src/Umbraco.Web/Models/Mapping/EntityMapDefinition.cs

```diff
@@ -186,7 +186,7 @@ private static void Map(ISearchResult source, SearchResultEntity target, MapperC
                 target.Name = source.Values.ContainsKey($"nodeName_{culture}") ? source.Values[$"nodeName_{culture}"] : target.Name;
             }
 
-            if (source.Values.TryGetValue(UmbracoExamineIndex.UmbracoFileFieldName, out var umbracoFile))
+            if (source.Values.TryGetValue(UmbracoExamineIndex.UmbracoFileFieldName, out var umbracoFile) && umbracoFile.IsNullOrWhiteSpace() == false)
             {
                 if (umbracoFile != null)
```

Comment:
@kjac the nested if-condition checking umbracoFile not is redundant now with this change 😃

Reply:
@bjarnef Duh.. my bad! I removed it but forgot to push before removing it 😂  I'll remove it now, thanks! 👍

Reply:
Fixed in https://github.com/umbraco/Umbraco-CMS/commit/e390820f2e1719c54d3521a77a8201b4d5062ed2?w=1 - thanks for noticing @bjarnef! ⭐